__tests__/createBrowserLikeFetch.spec.js

```diff
@@ -46,6 +46,29 @@ describe('createCookiePassingFetch', () => {
     expect(setCookie.mock.calls[0][2].maxAge).toEqual(3600000);
   });
 
+  it('does not add missing propterties', async () => {
+    const mockFetch = jest.fn(() => Promise.resolve({
+      headers: new Headers({
+        'set-cookie': [
+          'sessionId=1234rlakjhf; Domain=example.com; Path=/path/; HttpOnly;',
```

Comment:
should we add `"maxAge":null` here so it is clear that this tests is making sure it removes it?

Reply:
this test was written around the failing state.  `"maxAge":null` would not be a valid cookie string

Reply:
I just realised that the cookie string is correct the parsing is what was adding the null property, strange because this wasn't happening in previous versions 🤔